### ISSUE TITLE
Failed assertion, Read-only fields can't have autofill hints.

### ISSUE THREAD

**Issue by sm2017** [adar2378/pin_code_fields#114]
I have the following error in the console , but app is not crashed

```
════════ Exception caught by widgets library ═══════════════════════════════════════════════════════
Read-only fields can't have autofill hints.
'package:flutter/src/widgets/editable_text.dart':
Failed assertion: line 488 pos 10: '!readOnly || autofillHints == null'
The relevant error-causing widget was: 
  PinCodeTextField
```

```dart
PinCodeTextField(
  appContext: context,
  length: widget._codeLength,
  enabled: !(state is FormBlocSubmitting),
  obscureText: false,
  animationType: AnimationType.fade,
  animationDuration: Duration(milliseconds: 300),
  backgroundColor: Colors.transparent,
  pinTheme: PinTheme(
    shape: PinCodeFieldShape.underline,
    borderRadius: BorderRadius.circular(5),
    fieldHeight: 50,
    fieldWidth: 40,
    activeColor: Palette.nearlyBlack,
    selectedColor: Theme.of(context).primaryColor,
    inactiveColor: Palette.grey,
  ),
  keyboardType: TextInputType.number,
  controller: _textEditingController,
  onChanged: (value) {
    verificationFormBloc.code.updateValue(value);
  },
  onCompleted: (value) {
    verificationFormBloc.code.updateValue(value);
    context.bloc<VerificationFormBloc>().submit();
  },
),
```

**Comment by adar2378** [comment]
Thank you, will check this

**Comment by sm2017** [comment]
@adar2378 Do you check it?

**Comment by adar2378** [comment]
Hi, I tried but could not produce the warning. I will try again.

**Comment by adar2378** [comment]
I fixed that in the latest update, check 6.0.1
